packages/flutter_android_lifecycle/pubspec.yaml

```diff
@@ -0,0 +1,60 @@
+name: flutter_android_lifecycle
+description: A new flutter plugin project.
+version: 0.0.1
+author:
+homepage:
+
+environment:
+  sdk: ">=2.1.0 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+
+# For information on the generic Dart part of this file, see the
+# following page: https://dart.dev/tools/pub/pubspec
+
+# The following section is specific to Flutter.
+flutter:
+  # This section identifies this Flutter project as a plugin project.
+  # The androidPackage and pluginClass identifiers should not ordinarily
+  # be modified. They are used by the tooling to maintain consistency when
+  # adding or updating assets for this project.
+  plugin:
+    androidPackage: com.example.flutter_android_lifecycle
+    pluginClass: FlutterAndroidLifecyclePlugin
```

Comment:
Also - make sure the apps using this plugin can build and run for iOS.
Note that we should be soon able to delete all ios code from this plugin once https://github.com/flutter/flutter/issues/39659 is resolved and we migrate to the new pubspec schema, but that will have to wait for the next stable release.

Reply:
Is there a process for verifying that?

Reply:
build and run the example app on iOS (or any app the depends on the plugin)

Reply:
@amirh @blasten I got the following error when attempting to build and run the iOS app. Do either of you happen to know a remedy?

```
Downloading ios tools...                                            6.2s
Downloading ios-profile tools...                                    4.4s
Downloading ios-release tools...                                   17.3s
Launching lib/main.dart on iPhone Xʀ in debug mode...
Running pod install...
Running Xcode build...
Xcode build done.                                           30.7s
Finished with error: ProcessException: Process exited abnormally:
An error was encountered processing the command (domain=NSPOSIXErrorDomain, code=22):
Failed to install the requested application
The application's Info.plist does not contain CFBundleVersion.
Ensure your bundle contains a CFBundleVersion with a valid semantic version number.
  Command: /usr/bin/xcrun simctl install 0B543D29-BF06-4FF5-951E-2B98933F2403 [redacted]/flutter_plugins/packages/flutter_android_lifecycle/example/build/ios/iphonesimulator/Runner.app
```

Reply:
It looks similar to https://github.com/flutter/flutter/issues/34477 cc @jmagman